CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+our project that we need help with including:
+
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Github Discussions
+* Web design
+* Communications / Social Media / Blog Posts
+* Release management
+
+Not everything happens through a GitHub pull request. Please [contact us](https://github.com/topolvm/topolvm/discussions)
+and let's discuss how we can work together.
+
+## Find an Issue
+
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. [good first issue](https://github.com/topolvm/topolvm/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) has extra information to
+help you make your first contribution. [help wanted](https://github.com/topolvm/topolvm/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3A%22help+wanted%22) are issues
+suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
+
+Sometimes there won’t be any issues with these labels. That’s ok! There is
+likely still something for you to work on. If you want to contribute but you
+don’t know where to start or can't find a suitable issue, you can ask for an
+issue to work on the [Discussions](https://github.com/topolvm/topolvm/discussions).
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it. Something like "I want to work on this" is fine.
+
+## Ask for Help
+
+The best way to reach us with a question when contributing is to ask on:
+
+* The original github issue
+
+## Pull Request Lifecycle
+
+When you submit a PR, it is not a draft and all tests passes, we will start
+a review process. Reviewers are selected automatically, so you do not care about.
+
+If Your change is approved by us, we will merge it immediately and it will be
+shipped on the next monthly release.
+
+When there has been no activity for 30 days, the stale bot will label it stale.
+And if there are no activity for another 7 days, it will be closed.
+
+## Development Environment Setup
+
+Our recommended environment is Ubuntu 20.04. Following steps modifies you system globally,
+we suggest to you preparing a dedicated physical or virtual machine.
+
+1. Download the repository.
+
+    ```console
+    git clone git@github.com:topolvm/topolvm.git
+    ```
+
+2. Install required tools.
+
+    ```console
+    cd topolvm
+    make setup
+    ```
+
+3. Make changes you wish
+
+4. Test your changes by end-to-end test
+
+    ```console
+    cd e2e
+    make start-lvmd
+    make test
+    ```
+
+## Sign Your Commits
+
+### DCO
```

Comment:
There are two options DCO and CLA. I choose DCO because it is lightweight.

Please provide your though, maintainers.

Reply:
I choose DCO too. I think CLA is high operating costs and we don't need as much rigor as the CLA.